### DISCUSSION TITLE
Experimentos utilizados no Front-End

### DISCUSSION THREAD

**Discussion by raphaklaus** [frontendbr/forum#25] [Discussão]
Olá pessoal, quais experimentos de front end vocês consideram incríveis? Fiquem à vontade para comentar também como foram feitos, quais bibliotecas foram usadas por exemplo...

**Comment by bernardodiasc** [comment]
Alex the CSS Husky: http://codepen.io/davidkpiano/pen/wMqXea
Apenas Sass e HTML (com uma pitada de SVG).

Uma parte que achei sagaz é como ele aplicou a animação em todos os elementos:

```
@mixin effect($name, $kf) {
  @keyframes #{$name} {
    @each $interval, $transform in $kf {
      #{percentage($interval/$total-kf)} {
        transform: $transform;
      }
    }
  }
}
```

**Comment by vitor-mendrone-deel** [comment]
Gosto muito desse Gameboy: 

http://bchanx.com/animated-gameboy-in-css

**Comment by vitor-mendrone-deel** [comment]
Outro que eu gosto bastante: 

http://species-in-pieces.com/

**Comment by lucasbento** [comment]
Cara, curto demais animações mas num manjo quase nada, vocês tem algum material para recomendar para um beginner nisso?

**Comment by vitor-mendrone-deel** [comment]
@lucasbsilva [Esse site](https://cssanimation.rocks/) tem alguns tutoriais interessantes pra ir pegando uma base. 

[Codepen](http://codepen.io/) também é um bom lugar pra encontrar coisas legais e dar uma xeretada nos códigos.

**Comment by felipefialho** [comment]
Vou colocar alguns que desenvolvi entre 2012 e 2014, posso? haha
- [Eric Cartman](http://codepen.io/LFeh/pen/qzDCJ)
- [Clockwork Orange](http://codepen.io/LFeh/pen/kjlnJ)
- [Merry Christmas!](http://codepen.io/LFeh/pen/EjcBK)
- [Piano Keyboard](http://www.felipefialho.com/piano/#.VrOtYvkrKHs)

**Comment by vitorbritto** [comment]
Modificando o título para pt-BR.

Cc: @raphaklaus @frontendbr/admins

**Comment by felipefialho** [comment]
@vitorbritto +1

**Comment by raphaklaus** [comment]
:+1:

**Comment by kazzkiq** [comment]
O [Tearable Cloth](http://codepen.io/dissimulate/pen/KrAwx) também é um exemplo muito show.

Eu também vou compartilhar dois meus :smile: _(se você espera algo nível Husky, nem entre haha)_:
[CSS Only Interactive Chart](http://codepen.io/kazzkiq/pen/NqgpLd)
[CSS Only Responsive Menu](http://codepen.io/kazzkiq/pen/pvqvxO)

**Comment by Cleysonlb** [comment]
Fiz um  superman Pixel por pixel e animei :) faz um tempo já mas ta ai!
[http://codepen.io/Cleyson/pen/oGnsb](http://codepen.io/Cleyson/pen/oGnsb)

**Comment by lagden** [comment]
Todos os meus mockups ficam no [codepen.io](http://codepen.io)!!
[codepen.io/lagden](http://codepen.io/lagden)

o último que fiz foi a tela do Matrix!!!
a ideia era fazer com **css**, só que tava dando um puta trampo!
http://codepen.io/lagden/full/PZBBaE

**Comment by raphaklaus** [comment]
@lagden Woww! De alguma forma a gente tava conectado esses dias então, auhauhauh, porque eu fiz esse mesmo efeito, mas utilizando só CSS.

http://raphaklaus.github.io/matrix-fx/

Eu vi seus outros pens, muito bom cara.

**Comment by lagden** [comment]
@raphaklaus caraca!!! ficou animal!! :astonished:

**Comment by adamsalves** [comment]
E aí pessoal, salve!

Esses dias fiz essa animação http://codepen.io/adams_alves/pen/Xdpaer Tem alguns errinhos, mas no geral ficou legal. ;)
